src/app/index.html

```diff
@@ -33,43 +33,170 @@
+        <path d="M10.9321 1532.12C223.52 1415.91 493.659 1442.34 711.724 1545.55C929.789 1648.76 1106.54 1812.99 1279.45 1972.9C1393.13 2078.01 1508.21 2186.26 1639.74 2280.75C1769.94 2374.22 1913.46 2452.41 2062.05 2507.24C2212.22 2563.1 2370.26 2594.98 2530.35 2601.7C2733.15 2610.1 2933.3 2575.43 3130 2524.69C3305.88 2479.43 3478.06 2421.68 3646.86 2364.97C3701.08 2346.71 3755.29 2328.47 3809.48 2310.25C3926.42 2270.88 4043.85 2231.31 4161.75 2187.86C4343.1 2121.05 4523.6 2043.46 4693.75 1963.42" stroke="#FCEC09" stroke-miterlimit="10"/>
+        <path d="M5.75684 1571.22C217.374 1452.16 487.363 1477.42 708.037 1577.47C928.711 1677.51 1111.11 1839.97 1292.39 1996.26C1410.45 2098.05 1531.84 2200.79 1664.2 2291.52C1795.23 2381.24 1935.63 2458.29 2081.5 2514.26C2229.34 2571.03 2382.93 2606.72 2542.6 2615.06C2744.6 2626.2 2943.67 2591.7 3139.27 2538.53C3313.93 2491.12 3485.46 2428.62 3653.56 2366.96C3707.58 2347.13 3761.59 2327.31 3815.58 2307.49C3932.02 2264.71 4048.87 2221.87 4165.87 2177.04C4345.79 2108.07 4525.3 2033.89 4699.64 1958.33" stroke="#FCEC09" stroke-miterlimit="10"/>
+        <path d="M0.539062 1610.01C211.186 1488.11 481.002 1512.21 704.285 1609.08C927.568 1705.96 1115.73 1866.87 1305.35 2019.5C1672.22 2314.61 2084.97 2596.96 2554.89 2628.1C2935.89 2653.36 3305.52 2509.76 3660.29 2368.66L4705.59 1952.86" stroke="#FCEC09" stroke-miterlimit="10"/>
+        </svg>
+        
+      <img
+        class="bike-img"
+        src="./images/yellow-bike.png"
+        alt="An image of a yellow bike"
+      />
+      <section class="header-info">
+        <img
+          width="131"
+          height="139"
+          src="./images/logo.png"
+          alt="A circular logo with wrenches and allen keys"
+        />
+        <h1 class="main-heading">
+          SCORPIO BIKE <br />
+          <span class="yellow-part">REPAIR</span> SHOP
+        </h1>
+        <p class="sub">
+          The <span class="yellow-part"> best </span> bike dealer and repairer
+          in Aarhus!
+        </p>
+        <section class="working-hours">
+          <section class="first-row">
+            <article>
+              <p>MON - THU</p>
+              <hr />
+              <p>8:30 - 5:00</p>
+            </article>
+
+            <article>
+              <p>FRI</p>
+              <hr />
+              <p>8:30 - 4:00</p>
+            </article>
+            <article class="desktop">
+              <p>SAT</p>
+              <hr />
+              <p>9:00 - 1:00</p>
+            </article>
+          </section>
+          <article class="second-row">
+            <p>SAT</p>
+            <hr />
+            <p>9:00 - 1:00</p>
+          </article>
+        </section>
+        <section class="action-buttons">
+          <button class="base-button white-button">
+            <svg
+              width="31"
+              height="31"
+              viewBox="0 0 31 31"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
```

Comment:
Class Icon should be put on the svgs. When inside base button it will put margin for the icon